.readthedocs.yaml

```
@@ -2,16 +2,14 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.9"
-  apt_packages:
-    - "graphviz"
-    - "default-jdk"
-sphinx:
-  configuration: docs/conf.py
-
-python:
-  install:
-  - requirements: docs/requirements.txt
-
-submodules:
-  include: all
+    nodejs: "18"
```

```suggestion
    nodejs: "22"
```

Version 18 is end of life, current LTS is 22.